android/src/main/java/com/rt2zz/reactnativecontacts/ContactsManager.java

```diff
@@ -460,7 +460,7 @@ public void openContactForm(ReadableMap contact, Callback callback) {
             imAddress.put(ContactsContract.Data.MIMETYPE, CommonDataKinds.Im.CONTENT_ITEM_TYPE);
             imAddress.put(CommonDataKinds.Im.DATA, imAccounts[i]);
             imAddress.put(CommonDataKinds.Im.TYPE, CommonDataKinds.Im.TYPE_HOME);
-            imAddress.put(CommonDataKinds.Im.PROTOCOL, CommonDataKinds.Im.CUSTOM_PROTOCOL);
+            imAddress.put(CommonDataKinds.Im.PROTOCOL, CommonDataKinds.Im.PROTOCOL_CUSTOM);
```

Comment:
[CUSTOM_PROTOCOL](https://github.com/AndroidSDKSources/android-sdk-sources-for-api-level-28/blob/master/android/provider/ContactsContract.java#L6708) - field that contains value of InstantMessaging service name
[PROTOCOL_CUSTOM](https://github.com/AndroidSDKSources/android-sdk-sources-for-api-level-28/blob/master/android/provider/ContactsContract.java#L6713) - data type to use custom service name

Reply:
![622868796780314645](https://user-images.githubusercontent.com/587438/87173488-d6f41680-c2a3-11ea-85ff-ec05cdb3feb7.png)